### PR TITLE
README.md consistency and misc adjustments.

### PR DESCRIPTION
**Pull request by TwinniDev**
being a readme gremlin as usual

### CODE REVIEW

**Review by kubo6472 — CHANGES_REQUESTED** [review]
telEgram, please :D

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
 ***Yes, Gabe. SteamOS functions well on a toaster.***
 
 This project attempts to bring the Steam Deck's Holo OS into a generic, installable format, and provide a close-to-official SteamOS experience.
 
-Click [here](https://t.me/HoloISO) to join **HoloISO** Telegram update channel;
+**HoloISO social media**
 
-Click [here](https://steamdeck.community/forums/holoiso.29/) to visit **HoloISO** discussion on Steam Deck Community forums
+- [Telgram](https://t.me/HoloISO) update channel.
```
Comment (olegshulyakov) [inline README.md:13]: Misprint, should be `Telegram`

**Comment by olegshulyakov** [comment]
@TwinniDev Please add section "HOW TO: Update":
1. Run "Check for updates" in DeckUI
2. `steamos-update check && steamos-update now`
3. `sudo pacman -Sy holostaging/holoiso-updateclient`

File: README.md [hunk 2]
```diff
@@ -93,9 +93,8 @@ Building the ISO:
 -
 Trigger the build by executing:
 ```
-pacman -Sy archiso
+sudo pacman -Sy archiso
```
Comment (getchoo) [inline README.md:96]: [don't use `pacman -Sy`](https://wiki.archlinux.org/title/System_maintenance#Partial_upgrades_are_unsupported)

Comment (olegshulyakov) [reply README.md:96]: @getchoo What is a proper way? `pacman -Syu`?

Comment (getchoo) [reply README.md:96]: nope just `pacman -S`

**Pull request closed without merging**